packages/core-ethereum/src/ethereum.ts

```diff
@@ -60,7 +60,7 @@ export async function createChainWrapper(
     wallet
   ) as HoprChannels
 
-  const genesisBlock = (await provider.getTransaction(hoprChannelsDeployment.transactionHash)).blockNumber
+  const genesisBlock = (await provider.getTransaction(hoprChannelsDeployment.transactionHash))?.blockNumber ?? 0
```

Comment:
Shouldn't we instead assert that we have a valid block number at this point? If we cannot get the block number setting it to zero might just push the issue further down the logic and it will resurface later.

Reply:
These changes are temporary, will remove once this PR is not a draft (they help debug localhost)